pulp_2to3_migration/app/migration.py

```diff
@@ -273,6 +273,9 @@ def complex_repo_migration(plugin, pulp3_repo_setup, repo_name):
                     dist_migrator, progress_dist, dist,
                     migrated_repo.pulp3_repository_version
                 )
+                # add distirbutors specified in the complex plan
+                # these can be native and not native distributors
+                migrated_repo.pulp2_dists.add(dist)
```

Comment:
here, we will add all the distributors specified in the complex migration plan - whether native or not.
Although, at this point a native relation has been established in the pre_migration step.